src/rpm/builder.rs

```diff
@@ -951,6 +965,22 @@ impl RPMBuilder {
             ));
         }
 
+        if let Some(vendor) = self.vendor {
+            actual_records.push(IndexEntry::new(
+                IndexTag::RPMTAG_VENDOR,
+                offset,
+                IndexData::StringTag(vendor),
+            ));
+        }
+
+        if let Some(repository) = self.repository {
+            actual_records.push(IndexEntry::new(
+                IndexTag::RPMTAG_URL,
```

Comment:
Can you explain this?  URL is typically a website associated with the actual software the package is building, not the repository it came from.  Accordingly I'm not sure I understand where the "repository" stuff is coming from, why it's named that way, etc.

example packages from Fedora

```
[dalley@thinkpad ~/]$ dnf info firefox

Installed Packages
Name         : firefox
Version      : 108.0.1
Release      : 3.fc36
Architecture : x86_64
Size         : 271 M
Source       : firefox-108.0.1-3.fc36.src.rpm
Repository   : @System
From repo    : updates
Summary      : Mozilla Firefox Web browser
URL          : https://www.mozilla.org/firefox/
License      : MPLv1.1 or GPLv2+ or LGPLv2+
Description  : Mozilla Firefox is an open-source web browser, designed for standards
             : compliance, performance and portability.

[dalley@thinkpad ~/]$ dnf info zlib

Name         : zlib
Version      : 1.2.11
Release      : 33.fc36
Architecture : x86_64
Size         : 199 k
Source       : zlib-1.2.11-33.fc36.src.rpm
Repository   : @System
From repo    : updates
Summary      : Compression and decompression library
URL          : https://www.zlib.net/
License      : zlib and Boost
Description  : Zlib is a general-purpose, patent-free, lossless data compression
             : library which is used by many different programs.

```

Reply:
I have changed from repository to url and added a vcs option. Currently none of the test assets have any VCS information so cannot test against those explicitly.

Reply:
Ack.  Could you add a TODO to that test, in that case?